packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -98,6 +98,64 @@ enum DragAnchor {
 /// widget always displays [child].
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=QzA4c4QHZCY}
+/// 
+/// {@tool dartpad --template=stateless_widget_material}
+///
+/// The follwing example has the [Draggable] widget along with the [DragTarget] in a row demonstrating what happens when you drag the element to the target.
```

Comment:
Prefer lines < 80 characters. Also, fixed some typos and added a little clarification:
```suggestion
/// The following example has a [Draggable] widget along with a [DragTarget] 
/// in a row demonstrating an incremented `acceptedData` integer value when 
/// you drag the element to the target.
```